README.md

```diff
@@ -275,6 +275,7 @@ Install the following components:
 - [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk)
 - [Visual Studio 2015](https://www.visualstudio.com/downloads)
 - [Clang 6](http://releases.llvm.org/download.html)
+- [Python 3.7](https://www.python.org/ftp/python/3.7.0/python-3.7.0b1.exe)
```

Comment:
Point to the final release: https://www.python.org/ftp/python/3.7.0/python-3.7.0.exe